node.gyp

```diff
@@ -2,6 +2,8 @@
   'variables': {
     'v8_use_siphash%': 0,
     'v8_trace_maps%': 0,
+    'v8_enable_pointer_compression%': 0,
+    'v8_enable_31bit_smis_on_64bit_arch%': 0,
```

Comment:
Isn't there a risk a conflict with the lines in common.gypi? I'm thinking about native add-ons that must be built with the same options as node.

Reply:
What do you mean? Can you expand a bit?

Reply:
Both variables are set to zero in common.gypi. If this change overrides the values, we can remove the lines there. If it doesn't, I wonder if add-ons will always be built without pointer compression enabled

Reply:
Actually, our addon tests probably fail because they get the values from common.gypi instead of the configure flag.

Reply:
You were right! all native addons tests are now passing!